src/ray/gcs/gcs_server/gcs_actor_scheduler.cc

```diff
@@ -47,9 +47,10 @@ void GcsActorScheduler::Schedule(std::shared_ptr<GcsActor> actor) {
   RAY_CHECK(actor->GetNodeID().IsNil() && actor->GetWorkerID().IsNil());
 
   // Select a node to lease worker for the actor.
-  const auto &node = SelectNode(actor);
+  const auto &node_id = SelectNode(actor);
```

Comment:
Please help change it to `auto node_id`. I think the previous one is not correct.
SelectNode returns NodeID, so the reference here doesn't make sense.